### DISCUSSION TITLE
Stirling PDF update not working

### DISCUSSION THREAD

**Discussion by Seqway** [tteck/Proxmox#3988] [Help Needed] [answered]
Hello,
i installed Stirling via the helper script.
It is mention to type in the bash xyz command into the LXC console. If i do this nothing happens. Same if i type in COMMAND.

Is there an error ? Can someone help me ?

**Comment by havardthom** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/5e4640d1-6ca1-4b2d-9cc7-eed592aa99d6)

Works fine here. Either run `bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/stirling-pdf.sh)"` or type `update` in the LXC console.

**Comment by tteck** [comment]
> If i do this nothing happens.

🌐 It seems that you're facing a network problem, _probably_ linked to DNS.

**Comment by Seqway** [comment]
Per random i let the window open ... it took nearly 5 Minutes to get the upgrading screen ! It is working but needs really long to show screen.
And no i do have no DNS problem because apt update / upgrade runs fast and normal !
